### PR TITLE
SSKW-29 User profile page

### LINKED ISSUE #29: SSKW - User profile page
## Checklist ✅

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/Sphinx-Society/kaizen-web-client/issues) of this repository and believe that this is not a duplicate.

## Summary 💡

This page will be shown when an user enters to profile or when the administrator access to user profile. But it can change some atoms or organisms on it depending on the intention of entering and the role.

## Requirements 🌈

- If a non admin or admin user enters to the profile it will let the user sign out from ther
- If an admin user go to manage a user profile it will not show the sign out button
- It must follow the design system
- It must have documentation
- It must have tests
- It must be available in dark and light theme

## Motivation 🔦

This page will let the users and the admin manager the users profiles.

### PR DESCRIPTION
**Pull request by jhnemogap**
## Checklist ✅

- [x] I have followed the [contributor guideline](https://github.com/Sphinx-Society/kaizen-web-client/blob/master/CONTRIBUTING.md)
- [x] Tests are included
- [x] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Sphinx-Society/kaizen-web-client/blob/master/CONTRIBUTING.md/#Commits)

resolves #29

### CODE REVIEW

File: src/client/redux/user/user.actions.requests.js [hunk 1]
```diff
@@ -48,29 +49,192 @@ export const listUsers = (page = 1, documentId, role) => async (dispatch) => {
...
+      firstName,
+      lastName,
+      birthDate,
+      phoneNumber: phone,
+      avatar,
+      gender,
+      country,
+      email,
+    },
+  };
```
Comment (omarefg) [inline src/client/redux/user/user.actions.requests.js:218]: From line 195 to 218 should be in the user service, the reducer should not have that responsibility.

File: src/client/utils/parseJwt.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+export const parseJwt = (token) => {
+  const base64Url = token.split('.')[1];
```
Comment (omarefg) [inline src/client/utils/parseJwt.js:3]: Are we going to use this function?

Comment (jhnemogap) [reply src/client/utils/parseJwt.js:3]: This is not used anywhere now, let's burn it

Comment (omarefg) [reply src/client/utils/parseJwt.js:3]: Yeah :guitar:

File: src/client/redux/user/user.actions.requests.js (hunk 1 above)
Comment (jhnemogap) [reply src/client/redux/user/user.actions.requests.js]: Modified, there were two other similar cases and they were changed.

**Pull request merged**